src/org/labkey/test/components/ui/domainproperties/EntityTypeDesigner.java

```diff
@@ -230,6 +232,27 @@ public Optional<WebElement> optionalWarningAlert()
         return elementCache().optionalWarningAlert();
     }
 
+    public String getStoredAmountDisplayUnits()
+    {
+        return elementCache().storedAmountDisplayUnitsSelect.getValue();
+    }
+
+    public T setStoredAmountDisplayUnits(String displayUnits)
+    {
+        elementCache().storedAmountDisplayUnitsSelect.select(displayUnits);
+        return getThis();
+    }
+
+    public boolean isBarcodeInfoShown()
+    {
+        return elementCache().uniqueIdMsgLoc.existsIn(this);
+    }
+
+    public String getBarcodeInfoMsg()
+    {
+        return elementCache().uniqueIdMsgLoc.waitForElement(this, WAIT_FOR_JAVASCRIPT).getText();
+    }
```

Comment:
Same comment here, these fields aren't used and are not visible outside of the app.

Reply:
I had figured it might be a useful check for future me to be able to confirm that from outside of the app but okay, consider it gone